src/main/java/net/dv8tion/jda/core/requests/restaction/order/ChannelOrderAction.java

```diff
@@ -84,6 +84,16 @@ public ChannelOrderAction(Guild guild, ChannelType type)
         this.orderList.addAll(chans);
     }
 
+    // Internal usage only.
+    // This is used as a superclass constructor call for CategoryOrderAction.
+    ChannelOrderAction(Guild guild, ChannelType type, Collection<T> channels)
+    {
+        super(guild.getJDA(), Route.Guilds.MODIFY_CHANNELS.compile(guild.getId()));
+        this.guild = guild;
+        this.type = type;
+        this.orderList.addAll(channels);
```

Comment:
Check if the collection is empty or null, check if all channels are from the correct type.

Reply:
While on that note, these checks are never made in the original constructor for `ChannelOrderAction`, should I add them there as long as I am doing this?

Reply:
See my comment above